src/ApprenticeAan/SFA.DAS.ApprenticeAan.Application/CalendarEvents/Queries/GetCalendarEvents/CalendarEvent.cs

```diff
@@ -1,4 +1,7 @@
-﻿namespace SFA.DAS.ApprenticeAan.Application.CalendarEvents.Queries.GetCalendarEvents;
+﻿using SFA.DAS.ApprenticeAan.Application.Entities;
+
+namespace SFA.DAS.ApprenticeAan.Application.CalendarEvents.Queries.GetCalendarEvents;
+
 public class CalendarEvent
```

Comment:
This class is used in search, so it should not be extended directly. You have two options, create derived class and add your additional properties there, or create a new class with all properties you need.

Reply:
I would have preferred CalendarEvent for this derive class name, however it is taken, so you can either take the pain of renaming this to CalendarEventSummary or name your new class using CalendarEventDetails.

Reply:
:-) No worries. Just so you know this class is named CalendarEventSummary in the FE